### PR TITLE
9.0 backport website form metada

### PR DESCRIPTION
**Pull request by chienandalu**
- [x] Depends on https://github.com/odoo/odoo/pull/24888

Website Form Metadata
=====================

Odoo has the hidden and unused ability to gather metadata from the website
forms and record into the record created from such form. This module unveils
it.

cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
The same about the README

**Review by yajo — APPROVED** [review]
I set to WIP to wait for upstream's merge, but module's OK.

**Review by rafaelbn — APPROVED** [review]
Hi @chienandalu @Yajo , I don't think @odoo will attend https://github.com/odoo/odoo/pull/24888 in v9. Could we propose it to OCA maybe? This is :+1:

**Comment by chienandalu** [comment]
Rebasing after Odoo's merge: https://github.com/odoo/odoo/pull/24888

**Comment by pedrobaeza** [comment]
Travis is red. Can you check?

**Comment by chienandalu** [comment]
There were some .rst lint errors: https://github.com/OCA/website/pull/480/files

**Comment by chienandalu** [comment]
All green now

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**
